tests/test_api.py

```diff
@@ -37,7 +38,7 @@
 
 
 @pytest.fixture
-def config(root="/tmp/gitman-shared"):
+def config(root=os.path.normpath(tempfile.gettempdir() + "/gitman-shared")):
```

Comment:
`pytest` has a `tmpdir` fixture (http://doc.pytest.org/en/latest/tmpdir.html) that should be used here instead of `tempfile.

Something like:

```
@pytest.fixture
def config(tmpdir, directory="gitman-shared"):
    root = str(tmpdir.join(directory))
    ...
```